app/timetables/tests/test_models.py

```diff
@@ -235,3 +236,37 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class VendorTest(TestCase):
+    """Test the Vendor model."""
+
+    def setUp(self):
+        Vendor.objects.create(
+            name='Spicy Foods',
+            info='Reach us at info@spicy-foods.com',
+            start_date=timezone.make_aware(timezone.datetime(2016, 7, 5, 13, 0, 0)),
+            end_date=timezone.make_aware(timezone.datetime(2017, 7, 5, 13, 0, 0))
+        )
+
+    def test_uniqueness_of_vendor_name(self):
+        vendor = Vendor(
+            name='spicy foods',
+            start_date=timezone.make_aware(timezone.datetime(2016, 8, 5, 13, 0, 0)),
+            end_date=timezone.make_aware(timezone.datetime(2017, 8, 5, 13, 0, 0))
+        )
+
+        self.assertRaises(ValidationError, vendor.save)
+
+    def test_vendor_end_date_is_later_than_start_date(self):
```

Comment:
@andela-eisaac: Got it. Thanks for the explanation. Would you say I really got it if the function name is changed to `test_enforcement_of_vendor_start_date_being_less_than_its_end_date`

and `test_uniqueness_of_vendor_name` changed to `test_enforcement_of_uniqueness_of_vendor_name`?